package.json

```diff
@@ -1,74 +1,65 @@
+  "private": true,
+  "version": "0.0.0",
   "type": "module",
-  "version": "0.0.1",
+  "license": "MIT",
   "scripts": {
-    "dev": "astro dev",
-    "build": "astro build",
-    "preview": "astro preview",
-    "astro": "astro",
-    "lint": "eslint . --ext .js,.mjs,.cjs,.ts,.tsx,.jsx,.astro",
-    "lint:fix": "npm run lint -- --fix",
-    "format": "prettier . --write",
-    "format:check": "prettier . --check",
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --report-unused-disable-directives",
+    "lint:fix": "eslint . --fix",
+    "preview": "vite preview",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "test": "vitest",
     "test:ui": "vitest --ui",
-    "test:run": "vitest run",
     "test:coverage": "vitest run --coverage",
     "test:watch": "vitest --watch"
   },
-  "dependencies": {
-    "@astrojs/react": "4.3.1",
-    "@emotion/cache": "11.14.0",
-    "@emotion/react": "11.14.0",
-    "@emotion/server": "11.11.0",
-    "@emotion/styled": "11.14.1",
-    "@mui/icons-material": "7.3.4",
-    "@mui/material": "7.3.4",
-    "astro": "5.13.9",
-    "react": "19.1.1",
-    "react-dom": "19.1.1"
-  },
   "engines": {
     "node": ">=20.3.0",
     "yarn": ">=1.22.21 <=1.22.22"
   },
+  "dependencies": {
+    "@emotion/react": "11.14.0",
+    "@emotion/styled": "11.14.0",
+    "@mui/icons-material": "6.3.0",
+    "@mui/material": "6.3.0",
+    "react": "18.3.1",
+    "react-dom": "18.3.1"
+  },
   "devDependencies": {
-    "@chromatic-com/storybook": "4.1.1",
-    "@eslint/js": "9.36.0",
-    "@storybook/addon-a11y": "9.1.10",
-    "@storybook/addon-docs": "9.1.10",
-    "@storybook/addon-vitest": "9.1.10",
-    "@storybook/react-vite": "9.1.10",
-    "@testing-library/jest-dom": "^6.9.1",
-    "@testing-library/react": "^16.3.0",
-    "@testing-library/user-event": "^14.6.1",
-    "@types/parse-json": "7.0.0",
-    "@types/prop-types": "15.7.15",
-    "@types/react": "19.1.13",
-    "@types/react-dom": "^19.2.0",
-    "@typescript-eslint/eslint-plugin": "8.44.0",
-    "@typescript-eslint/parser": "8.44.0",
-    "@vitejs/plugin-react": "^5.0.4",
-    "@vitest/browser": "3.2.4",
-    "@vitest/coverage-v8": "3.2.4",
-    "astro-eslint-parser": "1.2.2",
-    "eslint": "9.36.0",
+    "@chromatic-com/storybook": "3.2.2",
+    "@storybook/addon-essentials": "8.5.0",
+    "@storybook/addon-interactions": "8.5.0",
+    "@storybook/addon-links": "8.5.0",
+    "@storybook/addon-onboarding": "8.5.0",
+    "@storybook/blocks": "8.5.0",
+    "@storybook/react": "8.5.0",
+    "@storybook/react-vite": "8.5.0",
+    "@storybook/test": "8.5.0",
+    "@testing-library/jest-dom": "6.6.3",
+    "@testing-library/react": "16.1.0",
+    "@testing-library/user-event": "14.5.2",
+    "@types/react": "18.3.18",
+    "@types/react-dom": "18.3.5",
+    "@typescript-eslint/eslint-plugin": "8.20.0",
+    "@typescript-eslint/parser": "8.20.0",
+    "@vitejs/plugin-react-swc": "3.5.0",
```

Comment:
You added `@vitejs/plugin-react-swc` (line 49) but the Vite version pinned is `6.0.7` (line 62). Ensure compatibility between `@vitejs/plugin-react-swc` and your pinned Vite version; mismatches may break build/dev.

**Sugerencia:**
```typescript
Verify compatibility matrix between `vite` and `@vitejs/plugin-react-swc`. If you need the SWC plugin, test locally and bump/downgrade Vite accordingly. Alternatively, use `@vitejs/plugin-react` if you prefer the Babel/Refresh approach. Example: update Vite to a supported version for the plugin or use the standard plugin.
```